exporter/coralogixexporter/README.md

```diff
@@ -21,6 +21,7 @@ Example configuration:
 exporters:
   coralogix:
     # The Coralogix traces ingress endpoint
+    endpoint: "ingress.coralogix.com:443"
     traces:
       endpoint: "ingress.coralogix.com:443"
```

Comment:
Should we remove the traces, logs, and metrics endpoint?
I guess this can miss guide the users

Reply:
So we do need to provide support per telemetry, but I don't think that using endpoint is the right way.

I suggest moving to the telemetry-exporter model where we only setup domain (e.g. coralogix.com or private.coralogix.com) and we get the endpoint abstracted